bootstrap/README.md

```diff
@@ -27,7 +27,14 @@ STABILITY_THRESHOLD=<desired stability threshold>
 NETWORK=<mainnet or testnet>
 ```
 
-## 3. Download the Bitcoin state
+## 3. Checkout the working revision
+See [Proposal](https://dashboard.internetcomputer.org/proposal/94253)
+
+```
+git checkout fdfced51d002d7f16908642f29216a869ecd3627
+```
```

Comment:
The steps here are not specific to computing the state we did in this proposal, so I'd remove this. To validate that specific proposal, you should follow the instructions in the [launch branch](https://github.com/dfinity/bitcoin-canister/tree/launch/bootstrap).

Reply:
The launch proposal has relevant updates, e.g. uploader canister accepts hardcoded list of hashes. Otherwise one needs to pass more hashes than possible to shove into dfx call as startup flags, which I failed to do.